iuf

```diff
@@ -577,15 +577,14 @@ def main():
                 config.activity.abort_activity(background_only=True)
                 print(f"Use `{script_name} -a {activity} resume` to re-connect")
                 print("Bye!")
-                sys.exit(0)
 
         elif answer in ["f", "force"]:
             answer_text = "Forcing an immediate abort"
             already_answered = True
             config.args["force"] = True
             config.activity.abort_activity()
             print("Bye!")
-            sys.exit(0)
+
```

Comment:
We don't need to explicitly exit.  This along with a sleep 5 after collecting the threads seems to get rid of that threading error when exiting.